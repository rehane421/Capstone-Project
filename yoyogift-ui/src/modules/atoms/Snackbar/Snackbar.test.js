import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import Snackbar from "./Snackbar";

it("should render Snackbar", () => {
  let wrapper = shallow(<Snackbar />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
