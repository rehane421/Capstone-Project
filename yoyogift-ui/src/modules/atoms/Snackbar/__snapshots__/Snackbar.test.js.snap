// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Snackbar 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Snackbar))
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "bottom",
      }
    }
    open={true}
  >
    <WithStyles(ForwardRef(SnackbarContent))
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    />
  </WithStyles(ForwardRef(Snackbar))>
</Fragment>
`;
